scripts/core/auth/basic-auth-adapter.js

```diff
@@ -17,4 +17,16 @@
                     return response.data;
                 }));
             };
+
+
+            this.authenticateXmpp = function(jid, transactionId) {
```

Comment:
I know this method can be self-explanatory by name, but it'd be nice to document it for the sake of consistency.